app/src/test/java/com/github/se/cyrcle/model/address/AddressRepositoryNominatimTest.kt

```diff
@@ -139,7 +140,7 @@ class AddressRepositoryNominatimTest {
     setupMockResponse(mockResponse)
 
     var resultAddress: Address? = null
-    addressRepositoryNominatim.search("0.0,0.0", { address -> resultAddress = address }, {})
+    addressRepositoryNominatim.search("0.0,0.0", { address -> resultAddress = address.first() }, {})
```

Comment:
I think it makes sense, but I'd still like to verify, but why is the first always chosen?

Reply:
Because when asking for a specific point, Nominatim will answer you with only one adress hence why the list has 1 element and I get it by using `first()`.